conf/inria.yaml

```diff
@@ -0,0 +1,30 @@
+program:
+  overwrite: True
+
+  
+trainer:
+  gpus: 1
+  min_epochs: 5
+  max_epochs: 100
+  benchmark: True
+  log_every_n_steps: 2
+
+experiment:
+  task: "inria"
+  name: "inria_test"
+  module:
+    loss: "ce"
+    segmentation_model: "unet"
+    encoder_name: "resnet18"
+    encoder_weights: "imagenet"
+    learning_rate: 1e-3
+    learning_rate_schedule_patience: 6
+    in_channels: 3
+    num_classes: 2
+    ignore_zeros: True  # class 0 not used for scoring
+  datamodule:
+      root_dir: "data/inria"
+      batch_size: 2
```

Comment:
This is pretty small... Note that `conf/inria.yaml` should contain the optimal hyperparameters while `tests/conf/inria.yaml` should contain the bare minimum to get the tests to run.

Reply:
Added `tests/conf/inria.yaml`

Reply:
Now that the small values required to make the tests run fast are in `tests/conf/inria.yaml`, should these values be increased? I guess it requires a hyperparam tuning to determine what the best hyperparams are.